### ISSUE TITLE
`legit install` output has unreasonable newlines

### ISSUE THREAD

**Issue by wkentaro** [kennethreitz/legit#168]
```
% legit install
The following git aliases have been installed:

  git                  !legit branches
  branches
  git graft            !legit graft
  git                  !legit harvest
  harvest
  git                  !legit publish
  publish
  git                  !legit unpublish
  unpublish
  git                  !legit sprout
  sprout
  git sync             !legit sync
  git                  !legit switch
  switch
  git                  !legit resync
  resync
```

it is no need to add new line add git, I think this is a bug of clint. (because git XXX column is colored string).

**Comment by jpiper** [comment]
I have pushed this fix as 0.5.1 in clint, so this can probably be closed now.

**Comment by wkentaro** [comment]
Thanks!
